app/models/process_checklist.rb

```diff
@@ -0,0 +1,9 @@
+class ProcessChecklist < ActiveRecord::Base
+  translates :text
+  belongs_to :booking_type
+  has_many :orders
```

Comment:
Do we need a `through` here in order to use the join table?

Reply:
Unsure, seems to be working 😱